server/migrations/20190501142532_create_users_table.js

```diff
@@ -16,6 +16,7 @@ exports.up = knex =>
             .defaultTo('patron');
         table.timestamps();
         table.string('email_confirm_code');
+        table.jsonb('settings');
```

Comment:
we agreed as from now on, any new changes to a table should be made through a new migration.

Reply:
resolved now